internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -661,6 +677,47 @@ components:
             Name of the uploaded image. It must be unique in the given resource group.
             If name is omitted from the request, a random one based on a UUID is
             generated.
+    OCIUploadOptions:
+      type: object
+      required:
+        - user
+        - tenancy
+        - region
+        - bucket_name
+        - namespace
+        - private_key
+        - fingerprint
+        - compartment
+      properties:
+        user:
+          type: string
+          example: 'oci1.user.oc1..aaaabbbbbccccdd'
+        tenancy:
+          type: string
+          example: 'tenancy'
+        region:
+          type: string
+          example: 'il-jerusalem-1'
+        bucket_name:
+          type: string
+          example: 'name-of-bucket'
+        namespace:
+          type: string
+          example: 'aabbccdd'
+        image_name:
+          type: string
+          example: 'name-of-image'
+        private_key:
+          type: string
+          example: 'ssh-key-...'
+        fingerprint:
+          type: string
+          example: 'ssh-key-fingerprint...'
```

Comment:
These examples aren't necessary, but if we keep them, they should probably be valid but obviously fake examples of a key and fingerprint

Reply:
ack, removing

Reply:
> This is all still looking good. Other than the dependency packaging for Fedora, it seems we also need to figure out the `GrantImageAccess()` question and we're done.
> 
> The comments below are mostly minor things (typos and small recommendations or questions).
> 
> A more important note about changes to old distros: We generally don't modify distro definitions for distribution versions that have already been released (e.g., RHEL 8.4 and 8.5). For one, we wont make releases into those distro versions anymore (except in cases of critical bugs).
> 
> Has this been discussed? Do we want to support RHEL 8.4 and RHEL 8.5 in OCI? If it's part of the requirements for this, we might need to discuss how it should be handled (if we haven't already).

I'm dropping `distro/rhel84`